pkg/command/command.go

```diff
@@ -41,23 +40,14 @@ type Status struct {
 }
 
 // New creates a new command from the provided arguments.
-func New(cmd ...string) *Command {
-	return NewWithWorkDir("", cmd...)
+func New(cmd string, args ...string) *Command {
+	return NewWithWorkDir("", cmd, args...)
 }
 
-// New creates a new command from the provided workDir and the command
+// NewWithWorkDir creates a new command from the provided workDir and the command
 // arguments.
-func NewWithWorkDir(workDir string, cmd ...string) *Command {
-	args := strings.Fields(strings.Join(cmd, " "))
-
-	command := func() *Command {
-		if len(args) == 0 {
-			return &Command{exec.Command(cmd[0])}
-		} else if len(args) > 1 {
-			return &Command{exec.Command(args[0], args[1:]...)}
-		}
-		return &Command{exec.Command(args[0])}
-	}()
+func NewWithWorkDir(workDir, cmd string, args ...string) *Command {
+	command := &Command{exec.Command(cmd, args...)}
 
 	if workDir != "" {
```

Comment:
nit: I think the `if` could be omitted here, we could just do `command.cmd.Dir = workDir` in any case.